src/plugins/auto/README.md

```diff
@@ -0,0 +1,55 @@
+# OpenVINO™ AUTO Plugin
+
+The main responsibility of the AUTO plugin is to provide a unified device that enables developers to code deep learning applications once and deploy them anywhere. 
+
+Other capabilities of the AUTO plugin:
+* Static device selection intelligently loads a network to one device or multiple devices
+* CPU acceleration to start inferencing while the target device is still loading the network
+* Model priority support for loading multiple networks to multiple devices
+
+The component is written in `C++`. If you want to contribute to the AUTO plugin, follow [the common coding style rules](../../../docs/dev/coding_style.md).
+
+## Key contacts
+
+In case of any questions, review and merge requests, contact the [AUTO Plugin maintainer group](https://github.com/orgs/openvinotoolkit/teams/openvino-ie-auto-multi-maintainers) group.
+
+## Components
+
+AUTO plugin follow OpenVINO™ plugin architecture and consists of several main compoents:
+* Plugin class
```

Comment:
Usually this section contains physical subcomponents of this component like doc, public API and etc.
For example: https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/paddle#components

I don't think that we need to put here that it contains plugin, executable network and etc. classes. These classes are main parts of each plugin